xbmc/music/tags/TagLoaderTagLib.cpp

```diff
@@ -423,7 +423,7 @@ bool CTagLoaderTagLib::ParseTag(ID3v2::Tag *id3v2, EmbeddedArt *art, MUSIC_INFO:
 
         // @xbmc.org ratings trump others (of course)
         if      (popFrame->email() == "ratings@xbmc.org")
-          tag.SetUserrating(popFrame->rating() / 51); //! @todo wtf? Why 51 find some explanation, somewhere...
+          tag.SetUserrating(popFrame->rating() / 51); // Scale is: 1 -> 51, 1.5 -> 76, 2 -> 102, ...
```

Comment:
Maybe the `@todo` should be left in so it's easier to find?

Reply:
I fixed the todo and explained why one needs to divide by 51.

Reply:
Okay